src/game_launcher.cpp

```diff
@@ -834,21 +834,15 @@ void game_launcher::start_wesnothd()
 		filesystem::write_file(config, filesystem::read_file(filesystem::get_wml_location("lan_server.cfg")));
 	}
 
+	LOG_GENERAL << "Starting wesnothd";
+	try
+	{
 #ifndef _WIN32
-	std::string command = "\"" + wesnothd_program +"\" -c \"" + config + "\" -d -t 2 -T 5";
+		bp::spawn(wesnothd_program, "-c", "\""+config+"\"", "-d");
 #else
-	// start wesnoth as background job
-	std::string command = "cmd /C start \"wesnoth server\" /B \"" + wesnothd_program + "\" -c \"" + config + "\" -t 2 -T 5";
-	// Make sure wesnothd's console output is visible on the console window by
-	// disabling SDL's stdio redirection code for this and future child
-	// processes. No need to bother cleaning this up because it's only
-	// meaningful to SDL applications during pre-main initialization.
-	SetEnvironmentVariableA("SDL_STDIO_REDIRECT", "0");
+		// start wesnoth as background job
+		bp::spawn("cmd", "/C", "start", "/B", wesnothd_program, "-c", "\""+config+"\"");
```

Comment:
This doesn't resolve the issue at all. You're still calling the shell to parse a command line. I think it should be fine to just delete the first two arguments and put `"start"` as the first argument, with the caveat that it probably needs to be the _actual path to `start.exe`_ rather than just the name.